homework-classes/Model.js

```diff
@@ -39,13 +39,13 @@
       this.notify(this.state);
     }
 
-    static fetchJSON(url) {
-      return fetch(url).then(res => {
-        if (!res.ok) {
-          return new Error(`HTTP ${res.status} - ${res.statusText}`);
-        }
-        return res.status === 200 ? res.json() : null;
-      });
+    static async fetchJSON(url) {
+      const response = await axios.get(url);
+      if (!response.status === 200) {
```

Comment:
As @eidosam pointed out in the lecture, you don't need this test here as **axios** does it for you.